index.html

```diff
@@ -10,18 +10,25 @@
     <title>Pirate Bartender</title>
 </head>
 
-<body>
-    <header>
-        <h1>Pirate Bartender</h1>
+<body class='body'>
+    <header class='header'>
+        <h1 class='header__text'>Pirate Bartender</h1>
+        <img class='header__logo' src="images/pirate-skull-100566393-large.png" alt="pirate logo">
     </header>
 
-    <main>
+    <main class='main'>
         <article>
             <section>
                 <form id="js-form">
                     <!--Populated by app.js-->
                 </form>
             </section>
+            <section>
+                <h2 class="hidden" id="js-drink-header">Your drink!</h2>
```

Comment:
Wow, good catch. I didn't realize I was doing that. Habit from coding in JavaScript with single quotes.